src/Symfony/Component/Serializer/Tests/Mapping/Loader/YamlFileLoaderTest.php

```diff
@@ -97,6 +97,8 @@ public function testSerializedName()
         $attributesMetadata = $classMetadata->getAttributesMetadata();
         $this->assertEquals('baz', $attributesMetadata['foo']->getSerializedName());
         $this->assertEquals('qux', $attributesMetadata['bar']->getSerializedName());
+        $this->assertEquals('duxi', $attributesMetadata['duux']->getSerializedName());
```

Comment:
```suggestion
        $this->assertSame('duxi', $attributesMetadata['duux']->getSerializedName());
```
Also for the other possible ones in this test class